src/checkout/NotificationSettingsForm.tsx

```diff
@@ -43,8 +43,9 @@ const NotificationSettingsForm: FC = () => {
             </Grid.Column>
             <Grid.Column widthSM={Columns.Three}>
               <FormInput
+                icon={IconFont.CurrencyUSD}
                 id="balanceThreshold"
-                label={`Amount (${MINIMUM_BALANCE_THRESHOLD} minimum)`}
+                label={`Limit ($${MINIMUM_BALANCE_THRESHOLD} minimum)`}
```

Comment:
If check to see where the minimum is being referenced (I think the billing page) and see if it makes sense to add this there too

Reply:
@asalem1  

<img width="1728" alt="Screen Shot 2022-04-26 at 1 20 05 PM" src="https://user-images.githubusercontent.com/10736577/165385467-23a2ecaa-6bdc-41e5-8817-d6a3f08b04bb.png">

Reply:
$$